acinclude.m4

```diff
@@ -33,7 +33,7 @@ AC_DEFUN([CURL_CHECK_DEF], [
   AC_REQUIRE([CURL_CPP_P])dnl
   OLDCPPFLAGS=$CPPFLAGS
   # CPPPFLAGS comes from CURL_CPP_P
-  CPPFLAGS="$CPPPFLAGS"
+  #CPPFLAGS="$CPPPFLAGS"
```

Comment:
So  this effectively disables the `CURL_CPP_P` check that we added in ecf953432d86058. If you want to remove the assignment, then remove it. If you want to remove it, then why figure out `CPPPFLAGS` to begin with? If you can make the script work without the `CPPPFLAGS` logic properly, please elaborate on how and why that works.